tests/scripts/run_python_model_tests.sh

```diff
@@ -59,4 +59,9 @@ if [ "$ARCH_NAME" != "wormhole_b0" ]; then
 
     # Unet Shallow
     WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv $TT_METAL_HOME/models/experimental/functional_unet/tests/test_unet_model.py
+
+    # Mamba
+    WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv $TT_METAL_HOME/models/demos/wormhole/mamba/tests/test_mamba_ssm.py -k "pretrained_weight_false"
```

Comment:
Is it possible we can refactor this test so that it's callable with just one file?